search-service/src/test/kotlin/com/egm/stellio/search/web/TemporalEntityHandlerTests.kt

```diff
@@ -46,6 +46,8 @@ class TemporalEntityHandlerTests {
     @Value("\${application.jsonld.apic_context}")
     val apicContext: String? = null
 
+    val context = "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/" +
```

Comment:
isn't it the same as the above `apicContext`?

Reply:
yes